src/status_im/transport/core.cljs

```diff
@@ -0,0 +1,66 @@
+(ns status-im.transport.core
```

Comment:
Paragraph ish docstring explaining purpose of transport family ns in general, concise version of what's in status communication protocol doc